process-engine/bpmn-studio
# ✨ Implement Simple Confirm Widget

## What did you change?

⏰ Waiting for https://github.com/process-engine/management_api_contracts/pull/15
⏰ Waiting for https://github.com/process-engine/management_api_client/pull/10
⏰ Waiting for https://github.com/process-engine/process_engine/pull/142
⏰ Waiting for https://github.com/process-engine/skeleton/pull/72

This PR implements the confirm widget.

### TODO:

- Update Version of skeleton-electron

## How can others test the changes?

- Open up the BPMN-Studio 
- Go to the Detail View of any diagram 
- Add a user task
	- Add a extension property with name "preferredControl" and value "confirm"
	- Add a form field of type boolean with a label
- Run the diagram
- Notice the confirm widget

## PR-Checklist

Please check the boxes in this list after submitting your PR:
